JSDemos/Demos/PivotGrid/ExcelJsHeaderAndFooter/Vue/App.vue

```diff
@@ -15,13 +15,52 @@
         :show-data-fields="true"
         :show-row-fields="true"
         :show-column-fields="true"
-        :show-filter-fields="false"
-        :allow-field-dragging="false"
+        :show-filter-fields="true"
+        :allow-field-dragging="true"
         :visible="true"
       />
       <DxFieldChooser :enabled="false"/>
       <DxExport :enabled="true"/>
     </DxPivotGrid>
+    <div class="options">
+      <div class="caption">Export Headers</div>
+      <div class="option">
+        <DxCheckBox
+          id="export-data-field-headers"
+          :value="exportDataFieldHeaders"
+          :on-value-changed="onExportDataFieldHeadersChanged"
+          text="Data Field"
+        />
+      </div>
+      {{ ' ' }}
+      <div class="option">
+        <DxCheckBox
+          id="export-row-field-headers"
+          :value="exportRowFieldHeaders"
+          :on-value-changed="onExportRowFieldHeadersChanged"
+          text="Row Field"
+        />
+      </div>
+      {{ ' ' }}
```

Comment:
Why you don't use `&nbsp;` here?)

Reply:
&nbsp; it is not the same that {{ ' ' }} or {{ '' }} in this case